vacasa/connect/connect.py

```diff
@@ -899,6 +899,81 @@ def add_reservation_guests(self,
         return self._post(url, json={'data': {'attributes': payload,
                                               'type': 'reservation-guest'}}, headers=headers).json()
 
+    def add_indentity_check(self,
+                            reservation_id: int,
+                            idology_check_id: str,
+                            unit_id: int,
+                            name: dict,
+                            address: dict,
+                            block_list_id: int = None,
+                            passed_initial_check: bool = None,
+                            differentiator_shown: bool = None,
+                            questions_quantity: int = None,
+                            questions_correct: int = None,
+                            challenge_quantity: int = None,
+                            challenge_correct: int = None,
+                            received_soft_fail: bool = None,
+                            received_hard_fail: bool = None,
+                            approved_for_checkout: bool = None,
+                            idology_check_message: str = None,
+                            ):
+        """
+        Adds an identity record into the monolith for idology check result
+        :param reservation_id:
+        :param idology_check_id:
+        :param unit_id:
+        :param name:
+        :param address:
+        :param block_list_id:
+        :param passed_initial_check:
+        :param differentiator_shown:
+        :param questions_quantity:
+        :param questions_correct:
+        :param challenge_quantity:
+        :param challenge_correct:
+        :param received_soft_fail:
+        :param received_hard_fail:
+        :param approved_for_checkout:
+        :param idology_check_message:
```

Comment:
no point in putting these docstring things in there if they're not filled out...